web/concrete/src/Sharing/SocialNetwork/ServiceList.php

```diff
@@ -34,7 +41,7 @@ public static function get()
         $return = array();
         foreach($services as $serviceArray)
         {
-            $o = new Service($serviceArray[0], $serviceArray[1], $serviceArray[2]);
+            $o = new Service($serviceArray[0], $serviceArray[1], $serviceArray[2], $serviceArray[3]);
```

Comment:
Won't this cause undefined offsets when there's not a 4th element in the array?
http://3v4l.org/veh19